titan/output.py

```diff
@@ -35,9 +37,10 @@ def setup_aggregates(params: ObjMap, reportables, classes: List[str]) -> Dict:
     if classes == []:
         base_stats = {
             "agents": 0,
-            "deaths": 0,
             "deaths_hiv": 0,
```

Comment:
instead of keeping this as deaths_hiv, should we have it also be over every exit type? keep it generic?